### PR TITLE
Code level documentation improved

### PR DESCRIPTION
**Pull request by alimranahmed**
- Added missing @throw tags in php docs of some methods
- Corrected return type of methods

### CODE REVIEW

File: src/ExtractTags.php [hunk 1]
```diff
@@ -51,7 +55,7 @@ public static function fromJob($job)
      * Determine the tags for the given array.
      *
      * @param  array  $data
-     * @return array
+     * @return \Illuminate\Support\Collection
```
Comment (bonzai) [inline src/ExtractTags.php]: `array`

File: src/ExceptionContext.php [hunk 2]
```diff
@@ -10,7 +10,7 @@ class ExceptionContext
      * Get the exception code context for the given exception.
      *
      * @param  \Throwable  $exception
-     * @return array
+     * @return \Illuminate\Support\Collection
```
Comment (bonzai) [inline src/ExceptionContext.php]: `array`

**Comment by taylorotwell** [comment]
Some of your docblocks aren't even correct?

**Comment by browner12** [comment]
all of the `@throws` tags were added in https://github.com/laravel/telescope/commit/f44b96dfb8294281a1eecb7b32568438053716ce

make sure you fetch and rebase off the upstream.

**Pull request closed without merging**